SimpleCom/SimpleCom.cpp

```diff
@@ -83,18 +84,20 @@ static void StdInRedirector(HWND parent_hwnd) {
 	CloseHandle(overlapped.hEvent);
 }
 
+#define READ_BUF_SIZE 4096
+
 /*
  * Entry point for stdout redirector.
  * stdout redirects serial (read op) to stdout.
  */
 DWORD WINAPI StdOutRedirector(_In_ LPVOID lpParameter) {
 	HANDLE hStdOut = GetStdHandle(STD_OUTPUT_HANDLE);
-	char buf;
+	std::vector<char> buf(READ_BUF_SIZE, 0x00);
```

Comment:
I don't want to use `vector` here because `buf` do not need to be resized dynamically. It is enough to use `char [READ_BUF_SIZE]`.

Reply:
The reason I used a vector was for safely allocating on the heap instead of on the stack.

Reply:
Why heap allocation is better than on the stack? I think the stack is enough to hold `READ_BUF_SIZE` and local variables. If it is not enough unfortunately, the process would crash due to stack guard page - there are no security issue.

Could you use char array for this?